include/dlaf/lapack/enum_output.h

```diff
@@ -37,4 +37,19 @@ inline std::ostream& operator<<(std::ostream& out, const Norm& norm) {
   return out;
 }
 
+inline std::ostream& operator<<(std::ostream& out, const lapack::MatrixType& mtype) {
+  switch (mtype) {
+    case lapack::MatrixType::General:
+      out << "General";
+      break;
+    case lapack::MatrixType::Lower:
+      out << "Lower";
+      break;
+    case lapack::MatrixType::Upper:
+      out << "Upper";
+      break;
+  }
+  return out;
+}
+
```

Comment:
IIRC, in lapackpp utils there should be `matrixtype2str` for this purpose.

Reply:
I used the `matrixtype2str` utility and it works fine. The only difference is that types are all lower case. Is that fine?
I did not change the previous lines regarding norms (to use `norms2str` utility) because output strings are different from those we've implemented, for example `"OneNorm"`< is represented as `"1"`. @rasolca @albestro which option do you prefer?